docs/api/web-frame-main.md

```diff
@@ -0,0 +1,91 @@
+# webFrameMain
+
+> Control web pages and iframes.
+
+Process: [Main](../glossary.md#main-process)
+
+## Methods
+
+These methods can be accessed from the `webFrameMain` module:
+
+```javascript
+const { webFrameMain } = require('electron')
+console.log(webFrameMain)
+```
+
+### `webFrameMain.fromId(processId, routingId)`
```

Comment:
I'm a little sad to add a new top-level module just for this one function.

Perhaps we could make this `webContents.frameFromId(...)`?

or, to follow `BrowserWindow.fromId()`, make it a static method on the `WebFrameMain` class?

Reply:
I like the latter suggestion of adding something like `WebFrameMain.fromId()`.

I noticed that APIs only seem to use this pattern when the class itself is constructable (with `new`). 
- `WebContents` doesn't have a constructor and it provides a top-level `webContents` API.
- `BrowserView` has a constructor and it provides static methods on the class.
- `BrowserWindow` has a constructor and it provides static methods on the class.

In this case, `WebFrameMain` doesn't have a constructor and it would provide static methods on the class. This seems unique, but if we're okay with it, I'll move forward with this design.

Reply:
hm, you're right. Let's be consistent, then.